templates/server/puppetserver/conf.d/web-routes.conf.erb

```diff
@@ -0,0 +1,10 @@
+web-router-service: {
```

Comment:
https://gist.github.com/brandonweeks/bc72082ab65c17b11396

Replacement template with puppetserver 1.x and 2.x support :)